src/depfile_parser.in.cc

```diff
@@ -55,25 +55,22 @@ bool DepfileParser::Parse(string* content, string* err) {
       re2c:indent:string = "  ";
 
       nul = "\000";
-      escape = [ \\#*[|];
+      escape = [ \t#];
```

Comment:
Why is "\t" included?

Reply:
To match the GCC behavior here as closely as possible:
https://github.com/gcc-mirror/gcc/blob/22a8377023d59cc01ab0a84a1df56d0e1336efa3/libcpp/mkdeps.c#L66